scripts/k3s-versions.go

```diff
@@ -0,0 +1,190 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"slices"
+	"sort"
+	"strconv"
+	"strings"
+
+	"golang.org/x/mod/semver"
+)
+
+// golang.org/x/mod/semver *requires* a leading 'v' on versions, and will add missing minor/patch numbers.
+var minimumVersion = "v1.21"
```

Comment:
This is both here and in the workflow yaml file; this will be confusing if we try to edit the value here and it doesn't get picked up in CI runs. Might be better to keep it in one place instead of two.

Reply:
I first had it only in the script, but then made it overridable from the CLI. I'm fine with removing it from the workflow and have it always run with the default. It is not going to change very often anyways.

Reply:
I'm happy with either, just not both :)

Reply:
I removed the setting from the workflow.